blog/disable-alerts-health-management-api.md

```diff
@@ -0,0 +1,151 @@
+---
+slug: Mute Alerts 
+title: "How to mute alerts during maintenance windows or scheduled backups?"
+description: "Disable or Mute Alerts during maintenance windows"
```

Comment:
We'll need to add a preview image for this, too. Just a line at line five that has:

image: [image url]

Reply:
I will put something up..